src/components/EditCoursePage/index.jsx

```diff
@@ -171,6 +171,7 @@ class EditCoursePage extends React.Component {
   }
 
   formatAdditionalMetadataFields(courseData) {
+    const variant_id = courseData.additional_metadata.variant_id ? courseData.additional_metadata.variant_id : null;
```

Comment:
note here that if the field is left blank, we will pass along `null` in the payload to discovery instead of an empty string (which is what we want)